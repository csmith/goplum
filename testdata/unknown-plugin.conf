check foo.random "check" {
    interval = 2s
    alerts = "foo"
}
