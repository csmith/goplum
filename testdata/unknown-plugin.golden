"error configuring check check: no plugin found with name foo"